python/instrumentation/openinference-instrumentation-dspy/tests/openinference/instrumentation/dspy/test_instrumentor.py

```diff
@@ -9,7 +9,6 @@
 
 import dspy
 import pytest
-from dsp.modules.cache_utils import CacheMemory, NotebookCacheMemory
 from dspy.primitives.assertions import (
```

Comment:
1 test is failing here due to https://github.com/stanfordnlp/dspy/commit/ae86009366c6bf6c8874078f07122cbcfe67fa4b These likely will be uncommented in future releases. IMHO, dspy maintainers shouldn't have done this but it is what it is.

Reply:
More info:. https://github.com/stanfordnlp/dspy/pull/1943 Since this is going to be removed anyway I think this test should be removed and re added once the new assertions come in.